x-pack/plugins/licensing/server/plugin.test.ts

```diff
@@ -80,4 +80,21 @@ describe('licensing plugin', () => {
 
     expect(licenseTypes).toEqual(['basic', 'gold', 'platinum']);
   });
+
+  test('provides a licensing context to http routes', async () => {
```

Comment:
I'm not sure of the best way to test the new context. This test is basically useless, so I'm open to ideas. I could write a functional test that stands up another plugin for the sole purpose of testing this, but that felt like overkill...

Reply:
It'd probably be easier to test if you export the context provider function separately from registering it. That way you could have this test to ensure it gets registered, and a separate test that verifies the behavior of the provider function

Reply:
Sounds good to me! Done in https://github.com/elastic/kibana/pull/46586/commits/477bd5b27f7259c2af20f2d5253d077224558325